test/edu-benefits/components/BenefitsSelectionFields.unit.spec.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import BenefitsSelectionFields from '../../../src/js/edu-benefits/components/BenefitsSelectionFields';
+
+describe('<BenefitsSelectionFields>', () => {
+  const data = {
+    benefitsRelinquished: {
+      value: ['chapter30'],
+      dirty: false
+    },
+    chapter33: true
+  };
+  const onStateChange = sinon.spy();
+
+  const tree = SkinDeep.shallowRender(
+    <BenefitsSelectionFields
+        data={data}
+        onStateChange={onStateChange}/>
+  );
+
+  it('should render a subsection for chapter 33', () => {
+    expect(tree.everySubTree('ErrorableRadioButtons').length).to.equal(1);
+  });
+  it('should render four checkboxes', () => {
```

Comment:
These tests you've written are awesome, but I wonder if this level of detail is necessary. It seems like our unit tests should be focused on testing key features and that components have particular traits they need in order to render, but not sure if it should be this tied to the business logic of the form. Definitely open to your thoughts here!